library/include/hipblaslt-ext.hpp

```diff
@@ -115,6 +115,7 @@ namespace hipblaslt_ext
             = 0; //!< The aux leading dimension. Only works if mode is set to aux related epilogues.
         int aux_stride
             = 0; //!< The aux batch stride. Only works if mode is set to aux related epilogues.
+        int scaling_ab_type = 0; //!< 0 is scalar, 1 is vector. Only works if DataTypeA = DataTypeB = FP8.
```

Comment:
would it impact ROCm backward combability testing?

Reply:
Even if I separate them to a and b, we can't guarantee.